test/Microsoft.AspNetCore.SignalR.Tests/EndToEndTests.cs

```diff
@@ -98,13 +99,14 @@ public static IEnumerable<object[]> MessageSizesData
         {
             get
             {
-                yield return new object[] { new String('A', 5)};
+                yield return new object[] { new string('A', 5)};
```

Comment:
at least one of the larger messages should not end at the buffer boundary. Otherwise you are not testing a condition where the last message part does not fill the buffer completely.